geode-redis/src/main/java/org/apache/geode/redis/internal/RegionProvider.java

```diff
@@ -51,4 +65,23 @@ public RegionProvider(InternalCache cache) {
   public Region<String, Object> getConfigRegion() {
     return configRegion;
   }
+
+  /**
+   * Validates that the value passed in is a power of 2 and that it is not greater than
+   * {@link #REDIS_SLOTS}
+   *
+   * @throws ManagementException if there is a problem with the value
+   */
+  protected static void validateBuckets(int buckets) {
```

Comment:
the name "validateBuckets" is a little misleading. I thought it was going to validate the actual buckets. Perhaps "validateBucketCount" would be better.

Reply:
Thanks - fixed.